content/exporters/supported-exporters/Node.js/Prometheus.md

```diff
@@ -31,26 +34,56 @@ npm install @opencensus/exporter-prometheus
 ```
 
 ## Creating the exporter
+To create the exporter, we'll need to:
+
+* Import and use the Prometheus exporter package
+* Define a namespace that will uniquely identify our metrics when viewed on Prometheus
+* Expose a port on which we shall run a `/metrics` endpoint
+* With the defined port, we'll need a Promethus configuration file so that Prometheus can scrape from this endpoint
+
 Now let's use the Prometheus exporter:
 
 ```js
-const { Stats } = require('@opencensus/core');
+const { globalStats } = require('@opencensus/core');
 const { PrometheusStatsExporter } = require('@opencensus/exporter-prometheus');
 
 // Add your port and startServer to the Prometheus options
 const exporter = new PrometheusStatsExporter({
   port: 9464,
-  startServer: false
+  startServer: true
 });
 
-// Our Stats manager
-const stats = new Stats();
-
 // Pass the created exporter to Stats
-stats.registerExporter(exporter);
+globalStats.registerExporter(exporter);
+```
 
-// Run the server
-exporter.startServer(function callback() {
-  // Callback
-});
+and then for our corresponding `prometheus.yaml` file:
+
+```shell
```

Comment:
Should this be `yaml` instead of `shell`?

Reply:
Done.